.gitignore

```diff
@@ -246,3 +246,5 @@ Release/
 /git.VC.db
 *.dSYM
 /contrib/buildsystems/out
+/.cproject
+/.project
```

Comment:
This change seems totally unrelated to what the commit message says. Please [split it out](https://git-scm.com/docs/git-rebase#_splitting_commits) into its own commit.